### DISCUSSION TITLE
ATtiny1626, USART0 in RS485 Mode with PortMux - does that work?

### DISCUSSION THREAD

**Discussion by mikrocoder** [SpenceKonde/megaTinyCore#1046] [Q&A]
Hi,

has anyone ever successfully operated the USART0 in RS485 Mode incl. PortMux (on pin PA1, 2, 4)?
Whether bare metal or with megaTinyCore.

**Comment by SpenceKonde** [comment]
Which RS485 mode? There are two. One of them is still here and absolutely should work.... One of them was dropped from all later parts. I'm unsure what exactly it's supposed to do but it it's what I think, the docs leave many critical  and acceptable behavior cannot be defined without more bits. 

If you;re doing the normal RS485, I would expect it to work. Might have to set PA4 output, and make sure nothing else has override on that pin. I haven't tried it in that specific mode but I expect it to work. It works on every part I've tried..

**Comment by mikrocoder** [comment]
Hi,

Are there two RS485 modes? I only read about one in the datasheet. I just wanted to ask if it works with Portmux. In another forum someone claims that it works without Portmux and not with Portmux. He thinks it is a bug in the ATtiny. However, because the user has not shown any reasonable code, it is not possible to check what he has really programmed. I don't have an ATtiny myself.
